### PR TITLE
Acknowledgments

### PR DESCRIPTION
**Pull request by alastc**
Updates from the group.

### CODE REVIEW

**Comment by stevefaulkner** [comment]
@alastc what is the threshold for inclusion in the acknowledgements?

File: acknowledgements/ag-wg-active.html [hunk 1]
```diff
@@ -48,15 +48,14 @@ <h2>Participants of the AG WG active in the development of this document:</h2>
...
 		<li>Stefan Johansson (Invited Expert)</li>
 		<li>Marc Johlic (IBM Corporation)</li>
 		<li>Rick Johnson (VitalSource | Ingram Content Group)</li>
 		<li>Crystal Jones (Microsoft Corporation)</li>
 		<li>Andrew Kirkpatrick (Adobe)</li>
 		<li>John Kirkwood (Invited Expert)</li>
 		<li>Jason Kiss (Department of Internal Affairs, New Zealand Government)</li>
-		<li>Maureen Kraft (IBM Corporation)</li>
 		<li>JaEun Ku (University of Illinois at Urbana-Champaign)</li>
 		<li>Patrick Lauke (TetraLogical)</li>
```
Comment (patrickhlauke) [inline acknowledgements/ag-wg-active.html:60]: Minor pedantic thing, but: any chance it could be changed to Patrick H. Lauke (with the middle initial)?

Comment (alastc) [reply acknowledgements/ag-wg-active.html]: Yep.

**Comment by alastc** [comment]
> @alastc what is the threshold for inclusion in the acknowledgements?

See the email to the list, but essentially “contributing” includes both content and reviews, so surveys and discussions are included even if you didn’t explicitely add/change words in the spec.

File: acknowledgements/ag-wg-active.html (hunk 1 above)
Comment (patrickhlauke) [reply acknowledgements/ag-wg-active.html]: wunderbar

**Pull request merged**